R/eval-tidy.R

```diff
@@ -383,15 +400,27 @@ new_data_mask <- function(bottom, top = bottom, parent = NULL) {
 
 #' @export
 names.rlang_data_pronoun <- function(x) {
-  names(unclass(x)$src)
+  src <- unclass(x)$src
```

Comment:
When is `src` not a data mask? Is this about compatibility tests?

Reply:
Yeah, tried to remain compatible with previous versions